setup.cfg

```diff
@@ -1,3 +1,50 @@
+[metadata]
+name = httpx
+version = attr: src.get_version
```

Comment:
Imho this should be a file (httpx/version.txt) and we use importlib_resources in `__version__` to grab it